crate/server/src/database/cached_sqlcipher.rs

```diff
@@ -381,12 +442,12 @@ impl Database for CachedSqlCipher {
     async fn is_object_owned_by(
         &self,
         uid: &str,
-        userid: &str,
+        owner: &str,
```

Comment:
keep user here? The result can be that the user is not an owner no?

Reply:
I've renamed it to be conform to the `Database` trait:

```
    /// Test if an object identified by its `uid` is currently owned by `owner`
    async fn is_object_owned_by(
        &self,
        uid: &str,
        owner: &str,
        params: Option<&ExtraDatabaseParams>,
    ) -> KResult<bool>;
```